src/coreclr/jit/compiler.h

```diff
@@ -8449,6 +8449,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
             clsHnd = gtGetStructHandleForHWSIMD(simdType, simdBaseJitType);
         }
 
+        if (clsHnd == NO_CLASS_HANDLE)
+        {
+            clsHnd = gtGetCanonicalStructHandleForSIMD(simdType);
```

Comment:
Do we want/need a comment explaining why the fallback is here?

Reply:
Added a comment, does it look good?

Reply:
LGTM